### PR TITLE
fix(TBD-11382): Inconsistent behavior of job execution for EMR 5.29

### PR DESCRIPTION
**Pull request by sponomarova**
…upgrade jersey libs

**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

**Review by lbourgeois — DISMISSED** [review]
LGTM

**Review by AlixMetivier — DISMISSED** [review]

File: main/plugins/org.talend.hadoop.distribution.emr5290/plugin.xml [hunk 1]
```diff
@@ -2402,11 +2405,11 @@
...
+            libraries="avro-1.8.2.jar;commons-cli-1.2.jar;commons-collections-3.2.2.jar;commons-configuration-1.6.jar;commons-io-2.4.jar;commons-lang-2.6.jar;commons-logging-1.1.3.jar;emr-metrics-client-2.1.0.jar;gson-2.2.4.jar;guava-11.0.2.jar;hadoop-auth-2.8.5-amzn-5.jar;hadoop-common-2.8.5-amzn-5.jar;hadoop-hdfs-2.8.5-amzn-5.jar;htrace-core-3.1.0-incubating.jar;httpclient-4.5.5.jar;httpcore-4.4.9.jar;jackson-core-asl-1.9.14-TALEND.jar;jackson-jaxrs-1.9.13.jar;jackson-mapper-asl-1.9.14-TALEND.jar;jackson-xc-1.9.13.jar;jersey-client-2.22.2.jar;jersey-server-2.22.2.jar;jersey-common-2.22.2.jar;jersey-guava-2.22.2.jar;jetty-util-6.1.26-emr.jar;log4j-1.2.17.jar;protobuf-java-2.5.0.jar;servlet-api-2.5.jar;slf4j-api-1.7.10.jar;slf4j-log4j12-1.7.10.jar;jersey-client-1.9.jar;hadoop-hdfs-client-2.8.5-amzn-5.jar;htrace-core4-4.0.1-incubating.jar">
```
Comment (CarlosTalend) [inline main/plugins/org.talend.hadoop.distribution.emr5290/plugin.xml:2412]: why is jersey-client kept as 1.9?

Comment (CarlosTalend) [reply main/plugins/org.talend.hadoop.distribution.emr5290/plugin.xml:2412]: Also in lines 893-895, 905-907, 1939, 2390, and 2404 (even if MapReduce isn't supported)

**Review by CarlosTalend — CHANGES_REQUESTED** [review]

File: main/plugins/org.talend.hadoop.distribution.emr5290/plugin.xml [hunk 2]
```diff
@@ -1895,7 +1895,10 @@
              <library id="slf4j-api-1.7.10.jar"/>
              <library id="slf4j-log4j12-1.7.10.jar"/>
              <library id="log4j-1.2.17.jar"/>
-             <library id="jersey-core-1.9.jar"/>
+             <library id="jersey-client-2.22.2.jar"/>
```
Comment (lbourgeois) [inline main/plugins/org.talend.hadoop.distribution.emr5290/plugin.xml:1898]: I think https://github.com/Talend/tbd-studio-se/blob/b41dd714e1d8ac816075d8394754ea52137eb066/main/plugins/org.talend.hadoop.distribution.emr5290/plugin.xml#L903 libraryNeeded can be removed

File: main/plugins/org.talend.hadoop.distribution.emr5290/plugin.xml (hunk 1 above)
Comment (sponomarova) [reply main/plugins/org.talend.hadoop.distribution.emr5290/plugin.xml:2412]: Agree. Will modify that.

File: main/plugins/org.talend.hadoop.distribution.emr5290/plugin.xml (hunk 2 above)
Comment (sponomarova) [reply main/plugins/org.talend.hadoop.distribution.emr5290/plugin.xml:1898]: @lbourgeois could you, please explain a little bit? 
I didn't get why <libraryNeeded in line 903 has to be removed. Thanks.

Comment (lbourgeois) [reply main/plugins/org.talend.hadoop.distribution.emr5290/plugin.xml:1898]: Sorry I meant all the libraryNeeded block containing jersey-core-1.9.jar as it is no longer used

**Pull request closed without merging**